fbchat/models.py

```diff
@@ -531,7 +519,6 @@ class ThreadType(Enum):
     USER = 1
     GROUP = 2
     PAGE = 3
```

Comment:
We should keep `ThreadType.ROOM`, just change it's value to `2`, like `ThreadType.GROUP`, for backwards compatibility